src/System.Memory.Polyfill/System/Stream.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.IO;
+
+namespace System
+{
+    public static partial class MemoryPolyfill
```

Comment:
StreamPolyFill instead?

Reply:
I think we want to minimize the number of extensions types.